docs/configuration.md

```diff
@@ -0,0 +1,146 @@
+---
+id: config
+title: Dapplet`s config
+---
+
+When creating a dapplet module, one of the important entities is the **dapplet's config**. It allows you to create and configure individual parameters for the module.
+
+For example: the network in which the dapplet will work, configure the monetization of the dapplet, change the url addresses of the server and the overlay part of the dapplet.
+
+The configuration is implemented through the `schema.json`.
+
+The schema follows the rules defined in http://json-schema.org/.
+The Dapplet settings UI is generated by [react-jsonschema-form](https://react-jsonschema-form.readthedocs.io/en/latest/usage/single/).
+
+In the production environment, we set up the **dapplet's config**, and specify the parameters that will be available in the browser through the extension.
+
+:::caution
+
+You can edit and create a **dapplet config** only from the module's working environment. User can use **dapplet config**, in extension.
+
+:::
```

Comment:
I think that it is not necessary block. We can change paremeters of the config using `set`, `remove`, and `clear` functions. User cannot use config. Just its implementation - settings.